dashboards/grafana-dashboard-insights-yuptoo-general.configmap.yaml

```diff
@@ -629,7 +629,7 @@ data:
                 "uid": "${datasource}"
               },
               "editorMode": "code",
-              "expr": "sum(kafka_consumergroup_group_lag{topic=\"platform-mq-stage.platform.upload.announce\", group=\"qpc-group\"})",
+              "expr": "sum(kafka_consumergroup_group_lag{topic=~\".*platform.upload.announce\", group=\"qpc-group\"})",
```

Comment:
nitpick - 

```suggestion
              "expr": "sum(kafka_consumergroup_group_lag{topic=~\"platform.upload.announce\", group=\"qpc-group\"})",
```

Reply:
Hello @kgaikwad, We need `.*` regex because topic name in stage env is `platform-mq-stage.platform.upload.announce`, so if we remove `.*` then query only works for production env and not for stage.

Reply:
+1, gotcha then it makes sense to me!